app/models/manageiq/providers/redhat/infra_manager/api_integration.rb

```diff
@@ -264,6 +264,12 @@ def raw_connect_v4(options = {})
       # Get the timeout from the configuration:
       timeout, = ems_timeouts(:ems_redhat, options[:service])
 
+      # The constructor of the SDK expects a list of certificates, but that list can't be empty, or contain only 'nil'
+      # values, so we need to check the value passed and make a list only if it won't be empty. If it will be empty then
+      # we should just pass 'nil'.
+      ca_certs = options[:ca_certs]
+      ca_certs = [ca_certs] if ca_certs
```

Comment:
Sorry if you've already covered this but if it is a comma separated string should you split it up here?  It looks like `OvirtSDK4::Connection` is expecting an array of strings https://github.com/oVirt/ovirt-engine-sdk-ruby/blob/master/sdk/lib/ovirtsdk4/connection.rb#L132-L136